Sources/Conduit/Auth/AuthorizationCodeGrant/OAuth2AuthorizationResponse.swift

```diff
@@ -18,14 +18,19 @@ public struct OAuth2AuthorizationResponse {
     /// callback. This is primarily used to prevent CSRF attacks.
     public let state: String?
 
+    /// The acess the user has given to the consuming application.
+    /// This value is returned as a comma separated list in the query string.
+    public let customParameters: [String: String]?
```

Comment:
Changing this to `[String: String]` and defaulting to an empty dictionary would simplify the structure & consuming callsites

Also, documentation needs to be updated